jdi-light-bdd/src/main/java/com/epam/jdi/bdd/stepdefs/WebPageSteps.java

```diff
@@ -0,0 +1,109 @@
+package com.epam.jdi.bdd.stepdefs;
```

Comment:
- add start/end line anchor (^/$ respectively)
- add (?:I |) at the beginning of action steps

Reply:
Resolved